scripts/sonar/sonar-scan.sh

```diff
@@ -13,10 +14,10 @@ SONAR_LOGIN=""
 SONAR_ORG="hootenanny"
 SONAR_PROJECT="hoot"
 SONAR_SOURCES="./hoot-core,./hoot-js,./hoot-rnd,./tbs,./tgs"
-SONAR_THREADS="$(nproc)"
+SONAR_THREADS="$(( $(nproc) * 2 ))"  # Sonar scan doesn't max out CPU, double the threads
 
 # Getting parameters from the command line.
-while getopts ":a:b:j:l:o:p:r:s:t:" opt; do
+while getopts ":a:b:dj:l:o:p:r:s:t:" opt; do
```

Comment:
Add the `-d` option for passing debug to `sonar-scanner`

Reply:
Are you missing a :

Reply:
I'm not.  The `:` following a letter for `getopts` means that the option will be followed by a parameter.  Since I just want the debug `-d` option without anything following it.  The `b` option has a `:` following it because of the usage `-b develop` to set the branch from GitHub.  It does look awkward with only one option not being followed by a semicolon but it is correct.